angular.json

```diff
@@ -150,5 +150,8 @@
     "@schematics/angular:directive": {
       "prefix": "app"
     }
+  },
+  "cli": {
+    "analytics": false
```

Comment:
is this change related to the PR?

Reply:
No, they are not related to this PR and are side products during the local test. Thanks for pointing it out!